app/forms/provider_interface/offer_wizard.rb

```diff
@@ -4,7 +4,7 @@ class OfferWizard
 
     STEPS = { make_offer: %i[select_option conditions check],
               change_offer: %i[select_option providers courses study_modes locations conditions check] }.freeze
-    NUMBER_OF_FURTHER_CONDITIONS = 4
+    MAXIMUM_NUMBER_OF_FURTHER_CONDITIONS = 20
```

Comment:
Rethinking whether this is true: Is the actual limit just on the number of conditions (standard + further)?

Reply:
Agreed to limit to 18 further conditions in Manage